src/error-display/error-display.tsx

```diff
@@ -95,7 +96,9 @@ export const ErrorDisplay = ({
                             data-id="error-display-description"
                         >
                             {typeof updatedAssets.description === "string" ? (
-                                <p>{updatedAssets.description}</p>
+                                <Text.Body>
```

Comment:
thinking to target `p` in the `DescriptionContainer` styled component and add the styling instead so that people who use the JSX method could enjoy the styling too.